CHANGELOG.md

```diff
@@ -20,236 +20,9 @@ Please see https://github.com/rapidsai/cucim/releases/tag/v22.04.00a for the lat
 
 ## 🚀 New Features
 
-1. Update cucim.skimage API to match scikit-image 0.19 ([#190](https://github.com/rapidsai/cucim/pull/190)) [@glee77](https://github.com/glee77)
```

Comment:
Should we be deleting this stuff or simply moving it?

Reply:
The same content (detailed information on new features) is in our release notes so wanted to simplify it in CHANGELOG.md

Reply:
okay, that makes sense